packages/integration-tests/src/connectors-mock.ts

```diff
@@ -0,0 +1,100 @@
+export const facebookConnectorId = 'facebook-universal';
+export const facebookConnectorTarget = 'facebook';
+export const facebookConnectorConfig = {
+  clientId: 'application_foo',
+  clientSecret: 'secret_bar',
+};
+
+export const aliyunSmsConnectorId = 'aliyun-short-message-service';
+export const aliyunSmsConnectorConfig = {
+  accessKeyId: 'access-key-id-value',
+  accessKeySecret: 'access-key-secret-value',
+  signName: 'sign-name-value',
+  templates: [
+    {
+      usageType: 'SignIn',
+      templateCode: 'template-code-value',
+    },
+    {
+      usageType: 'Register',
+      templateCode: 'template-code-value',
+    },
+    {
+      usageType: 'Test',
+      templateCode: 'template-code-value',
+    },
+  ],
+};
+
+export const twilioSmsConnectorId = 'twilio-short-message-service';
+export const twilioSmsConnectorConfig = {
+  accountSID: 'account-sid-value',
+  authToken: 'auth-token-value',
+  fromMessagingServiceSID: 'from-messaging-service-sid-value',
+  templates: [
+    {
+      content: 'This is for sign-in purposes only. Your passcode is {{code}}.',
+      usageType: 'SignIn',
+    },
+    {
+      content: 'This is for registering purposes only. Your passcode is {{code}}.',
+      usageType: 'Register',
+    },
+    {
+      content: 'This is for testing purposes only. Your passcode is {{code}}.',
+      usageType: 'Test',
+    },
+  ],
+};
+
+export const aliyunEmailConnectorId = 'aliyun-direct-mail';
+export const aliyunEmailConnectorConfig = {
+  accessKeyId: 'your-access-key-id-value',
+  accessKeySecret: 'your-access-key-secret-value',
+  accountName: 'noreply@logto.io',
+  fromAlias: 'from-alias-value',
+  templates: [
+    {
+      subject: 'register-template-subject-value',
+      content: 'Logto: Your passcode is {{code}}. (regitser template)',
+      usageType: 'Register',
+    },
+    {
+      subject: 'sign-in-template-subject-value',
+      content: 'Logto: Your passcode is {{code}}. (sign-in template)',
+      usageType: 'SignIn',
+    },
+    {
+      subject: 'test-template-subject-value',
+      content: 'Logto: Your passcode is {{code}}. (test template)',
+      usageType: 'Test',
+    },
+  ],
+};
+
+export const sendgridEmailConnectorId = 'sendgrid-email-service';
+export const sendgridEmailConnectorConfig = {
```

Comment:
this should using an imported type from the connector actually?

Reply:
Add an issue to refactor this: LOG-3829